### PR TITLE
SB-1475: test that ArtifactEntry.created stores both date and time

### LINKED ISSUE #1475: OrientDb stores java.util.Date without time
# Bug Description

As described in the subject, `orientDb` stores `java.util.Date` without time. For example, once stored `org.carlspring.strongbox.domain.ArtifactEntry#created` is always at midnight.

# Steps To Reproduce

1. Run `org.carlspring.strongbox.services.impl.ArtifactEntryServiceTest` and watch the logs

# Expected Behavior

`org.carlspring.strongbox.domain.ArtifactEntry#created` and other `java.util.Date` fields should be stored in database with proper time

### PR DESCRIPTION
**Pull request by evsikov**
# Pull Request Description

This pull request is 3 of 3 needed to close #1475.
1 of 3 is https://github.com/strongbox/liquibase-orientdb/pull/2
2 of 3 is https://github.com/strongbox/strongbox-db/pull/16

# Acceptance Test

* [ ] Building the code with `mvn clean install -Dintegration.tests` still works.
* [ ] Running `mvn spring-boot:run` in the `strongbox-web-core` still starts up the application correctly.
* [ ] Building the code and running the `strongbox-distribution` from a `zip` or `tar.gz` still works.
* [ ] The tests in the [`strongbox-web-integration-tests`](https://github.com/strongbox/strongbox-web-integration-tests/) still run properly.

# Questions

* Does this pull request break backward compatibility? 
  * [ ] Yes
  * [X] No

* Does this pull request require other pull requests to be merged first? 
  * [X] Yes, please see: this pull request description
  * [ ] No

### CODE REVIEW

File: strongbox-storage/strongbox-storage-api/src/test/java/org/carlspring/strongbox/services/impl/ArtifactEntryServiceTest.java [hunk 1]
```diff
@@ -31,6 +31,7 @@
 import org.springframework.util.CollectionUtils;
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.carlspring.strongbox.services.support.ArtifactEntrySearchCriteria.Builder.anArtifactEntrySearchCriteria;
+import static org.junit.jupiter.api.Assertions.*;
```
Comment (mslowiak) [inline strongbox-storage/strongbox-storage-api/src/test/java/org/carlspring/strongbox/services/impl/ArtifactEntryServiceTest.java]: We decided to switch on AssertJ in tests.
Please change assertions to AssertJ way.

Comment (mslowiak) [inline strongbox-storage/strongbox-storage-api/src/test/java/org/carlspring/strongbox/services/impl/ArtifactEntryServiceTest.java]: Please avoid using wildcard imports.
https://stackoverflow.com/questions/147454/why-using-a-wild-card-with-a-java-import-statement-bad

Comment (evsikov) [reply strongbox-storage/strongbox-storage-api/src/test/java/org/carlspring/strongbox/services/impl/ArtifactEntryServiceTest.java]: Done.

Comment (evsikov) [reply strongbox-storage/strongbox-storage-api/src/test/java/org/carlspring/strongbox/services/impl/ArtifactEntryServiceTest.java]: Got it. Done.

**Comment by carlspring** [comment]
@evsikov ,

Thanks for this pull request! Would you mind using rebase, over merge when syncing your branch with the upstream? You can find more about this [here on our wiki](https://strongbox.github.io/developer-guide/git/rebase-vs-merge.html).

**Comment by carlspring** [comment]
You can use the same pull request and just force push.

**Pull request closed without merging**